### PR TITLE
Support node 12

### PR DESCRIPTION
**Pull request by ankhyk**

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -1,20 +1,16 @@
 # Change Log - oav
 
-## 12/19/2022 3.3.0
+## 12/01/2022 3.2.4
```
Comment (raych1) [inline ChangeLog.md]:
```suggestion
## 01/30/2023 3.2.4
```

File: ChangeLog.md [hunk 2]
```diff
@@ -1,20 +1,16 @@
 # Change Log - oav
 
-## 12/19/2022 3.3.0
+## 01/30/2023 3.2.4
 
+- Generate Examples - Support specified generation rule
```
Comment (raych1) [inline ChangeLog.md:5]:
```suggestion
- Generate Examples - Support specified generation rule
- Revert node requirement to >=12
```

File: package.json [hunk 3]
```diff
@@ -73,14 +73,15 @@
...
-    "eslint-plugin-prettier": "^4.2.1",
+    "eslint": "^7.32.0",
+    "eslint-config-prettier": "^6.15.0",
+    "eslint-plugin-import": "^2.23.4",
+    "eslint-plugin-prettier": "^3.4.0",
     "handlebars": "^4.7.7",
     "jest": "^26.6.3",
     "jest-junit": "^13.2.0",
     "npm-run-all": "^4.1.5",
+    "prettier": "^2.6.2",
```
Comment (raych1) [inline package.json:84]: Added `prettier` reference back so that node version lower than 14 is supported without errors.

**Review by mikeharder — APPROVED** [review]

**Review by weshaggard — COMMENTED** [review]
I defer my approval to @mikeharder.

**Pull request merged**